tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -477,6 +479,7 @@ def test_pantsd_parse_exception_success(self):
     finally:
       rm_rf(test_path)
 
+  @unittest.skip("Pantsd is not expected to handle parallel runs anymore.")
```

Comment:
If these will be skipped rather than deleted, we should probably refer to a ticket about re-enabling parallel runs for v2.